cmd/tdex/market.go

```diff
@@ -200,6 +200,33 @@ var (
 		},
 		Action: marketUpdatePriceAction,
 	}
+
+	marketGetReportCmd = &cli.Command{
+		Name:   "report",
+		Usage:  "get market report",
+		Action: marketGetReport,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "start",
+				Usage: "fetch balances from specific time in the past, please provide end flag also",
+			},
+			&cli.StringFlag{
+				Name:  "end",
+				Usage: "fetch balances from specific time in the past til end date, use with start flag",
+			},
+			&cli.IntFlag{
+				Name: "predefined_period",
+				Usage: "time predefined periods:\n" +
+					"       1 -> last hour\n" +
+					"       2 -> last day\n" +
+					"       3 -> last month\n" +
+					"       4 -> last 3 months\n" +
+					"       5 -> year to date\n" +
+					"       6 -> all",
+				Value: 2,
+			},
+		},
```

Comment:
maybe better to let the user specify the time range as string? like `LAST_HOUR | last_hour` for instance? Then would be up to the cli to parse the string into the correct enum proto value.
In case the string is unknown we can either use a default value (`all`?) or return an error.
What do you think guys?

Reply:
> In case the string is unknown we can either use a default value (all?) or return an error.

that should behave like 0, so yeah all the window from start like all.

> maybe better to let the user specify the time range as string?

I do think it's fine this way being documented, but if @sekulicd is down for it sure